blog/announcements.xml

```diff
@@ -1,6 +1,139 @@
 <?xml version="1.0" encoding="utf-8" ?>
 
 <news>
+<item>
+    <pubDate>Wed, 31 May 11:00:00 UTC</pubDate>
+    <title id="nixos-23.05">
+      NixOS 23.05 released
+    </title>
+    <description>
+      <figure>
+        <a href="https://github.com/NixOS/nixos-artwork/blob/master/releases/23.05-raccoon/stoat.png">
+          <img class="inline" src="/logo/nixos-logo-23.05-raccoon-lores.png" alt="NixOS 23.05 Stoat logo" width="100" />
+        </a>
+      </figure>
+      <p>
+        Hey everyone, we are <a href="https://github.com/RaitoBezarius">Ryan Lahfa</a> and <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        the release managers for this stable release and we are very proud to announce the public availability of NixOS 23.05 “Stoat”.
+      </p>
+
+      <p>
+        This release will receive bugfixes and security updates for seven months (up until 2023-12-30).
+      </p>
+
+      <ul>
+        <li>
+          <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05">
+            Release manual
+          </a>
+          <ul>
+            <li>
+              <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05-highlights">
+                Highlights
+              </a>
+            </li>
+            <li>
+              <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05-internal">
+                Internal changes
+              </a>
+            </li>
+            <li>
+              <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05-version-updates">
+                Notable version Updates
+              </a>
+            </li>
+            <li>
+              <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05-incompatibilities">
+                Backward Incompatibilities
+              </a>
+            </li>
+            <li>
+              <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05-notable-changes">
+                Other Notable Changes
+              </a>
+            </li>
+            <li>
+              <a href="/manual/nixos/stable/release-notes.html#sec-release-23.05-new-services">
+                New Services
+              </a>
+            </li>
+          </ul>
+        </li>
+        <li><a href="/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="/download.html#download-nixos">ISO Image Downloads</a></li>
+      </ul>
+
+      <p>
+        The 23.05 release was made possible due to the efforts of <strong>1652</strong>
+        contributors, who authored <strong>30371</strong> commits since the previous release.
+        <br />
+        Our thanks go the contributors who also take care of the continued stability and
+        security of our stable distribution.
```

Comment:
```suggestion
        security of our stable release.
```
The wording is unusual, I suppose you mean patching the release branch? It's not clear though.